src/components/Book.css

```diff
@@ -0,0 +1,78 @@
+.Book {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 700px;
+  background-color: #fff;
+  margin: 0 auto 15px auto;
+  padding: 15px 25px;
+}
```

Comment:
- [ ] It would be so much better if the width of the books meet the same width as the original design. 

Your design 

![Capture](https://user-images.githubusercontent.com/64173189/135726411-cada160d-b3b9-4c0c-8a92-b91b24774f17.JPG)

The design you must follow 

![Capture-1](https://user-images.githubusercontent.com/64173189/135726412-414b4bc6-2a34-4fe5-829e-5f7b2da99f7a.JPG)


```suggestion
.Book {
  display: flex;
  justify-content: space-between;
  align-items: center;
  max-width: 75%;
  background-color: #fff;
  margin: 0 auto 15px auto;
  padding: 15px 25px;
}
```

Kindly fix this minor issue 🛠️ 😄